prometheus/README.md

```diff
@@ -5,3 +5,5 @@ This directory contains configuration for the [Prometheus](https://prometheus.io
 This directory is mounted into the `prometheus` container. After making your changes to this directory,
 simply `docker restart prometheus` for your changes to take effect (depending on your change, Prometheus
 may respond to it as soon as you save the file).
+
+You can add `_rules.yml` and `_targets.yml` files to this directory and they will be picked up automatically.
```

Comment:
Can you document here exactly which file would need to be created in order to retain the Docker metric scraping behavior from before:

```
  # Scrape Docker for metrics.
  # You will need to configure docker for this to work: https://docs.docker.com/config/thirdparty/prometheus/#configure-docker
  - job_name: "docker"
    static_configs:
      # For Linux, use this line:
      #- targets: ["localhost:9323"]
      #
      # For Mac, use this line:
      #- targets: ["docker.for.mac.host.internal:9323"]
      #
      # For Windows, use this line:
      #- targets: ["docker.for.win.localhost:9090"]
```

Reply:
done